.expeditor/config.yml

```diff
@@ -33,11 +33,16 @@ github:
   # allow bumping the minor release via label
   minor_bump_labels:
     - "Expeditor: Bump Minor Version"
+  # allow bumping the major release via label
+  major_bump_labels:
+    - "Expeditor: Bump Major Version"
   # Which Github branches to build Omnibus releases from, and what versions
   # (as determined by the value in the VERSION file) those branches are responsible
   # for building.
   release_branch:
     - master:
+        version_constraint: 4.*
+    - chefdk-3:
         version_constraint: 3.*
     - chefdk-2:
```

Comment:
Can we cut 2.x?

Reply:
Do we ever plan to release DK one last time with the latest openssl?